src/components/metadata/metadata.js

```diff
@@ -266,7 +267,7 @@ const MetaData = ({
                   </>
                 )}
               </dl>
-              {hasPlot && (
+              {hasPlot && showDatasetPreviews && (
```

Comment:
maybe you add it just here 

const hasPreview = showDatasetPreviews && metadata?.preview;